examples/arp.rs

```diff
@@ -0,0 +1,16 @@
+extern crate pnetlink;
+
+use pnetlink::packet::netlink::NetlinkConnection;
+use pnetlink::packet::route::neighbour::{Neighbour,Neighbours};
+
+fn print_neighbour(neighbour: &Neighbour) {
+    println!("{:?}", neighbour);
```

Comment:
Yes, familiarity, I want them to serve as an example of "this is how you implement %command% in rust using this library". If you want extra features, you can just rename your example :)